angular/src/app/components/asset-detail/asset-detail.component.ts

```diff
@@ -39,6 +41,12 @@ export class AssetDetailComponent implements OnInit {
         featureSource = featureSource.replace(/([^:])(\/{2,})/g, '$1/');
         this.featureSource = featureSource;
 
+        if (this.feature.assets.length && this.feature.assets[0].display_path) {
```

Comment:
looks good but can this be moved out of the try block (either above or below).

The try/except block currently is written to configure the featureSource and pointCloudSource.  Looks like if there is a feature with no assets (i.e. a simple point or polygon with an associated image or point cloud etc), then line 39 fails (e.g. `let featureSource = this.envService.apiUrl + '/assets/' + this.feature.assets[0].path;`) and the except then sets featureSource to null. but in this scenario, the `this.title` never gets set.

Reply:
Good catch - updated.